pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -27,19 +27,39 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 	"kubevirt.io/client-go/log"
 
-	"kubevirt.io/kubevirt/pkg/network/cache"
-	netsetup "kubevirt.io/kubevirt/pkg/network/setup"
 	netvmispec "kubevirt.io/kubevirt/pkg/network/vmispec"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/converter"
 )
 
-func hotplugVirtioInterface(vmi *v1.VirtualMachineInstance, dom cli.VirDomain, currentDomain *api.Domain, updatedDomain *api.Domain) error {
-	vmConfigurator := netsetup.NewVMNetworkConfigurator(vmi, cache.CacheCreator{})
+type vmConfigurator interface {
+	SetupPodNetworkPhase2(domain *api.Domain, networksToPlug []v1.Network) error
+}
+
+type virtIOInterfaceManager struct {
+	converterContext *converter.ConverterContext
```

Comment:
Is this field still being used?
I think it was removed in https://github.com/kubevirt/kubevirt/pull/9399.

Reply:
Good catch! 

Removed it; done.